src/api/app/services/meta_controller_service/request_validator.rb

```diff
@@ -0,0 +1,25 @@
+module MetaControllerService
+  class RequestValidator
```

Comment:
I still don't think that the class name reflects what this class is doing: ``MetaValidator`` would make it more clear I think.